server/routes/tasks.routes.js

```diff
@@ -6,7 +6,7 @@ const router = express.Router({ mergeParams: true });
 
 router.get("/", controller.getTasks);
 
-router.post("/", validators.validateTask, controller.createTask);
+router.post("/", controller.createTask);
```

Comment:
Why did you delete validator?

Reply:
I was debugging it, and temporarily removed it. Forgot to put it back